pandas/tests/indexes/test_multi.py

```diff
@@ -1425,8 +1448,6 @@ def test_equals_multi(self):
 
         self.assertFalse(self.index.equals(self.index[:-1]))
 
-        self.assertTrue(self.index.equals(self.index._tuple_index))
```

Comment:
Still test this using `pd.Index(self.index.values)` instead of `self.index._tuple_index`? (unless this is covered somewhere else to test equality of MI against object Index of tuples).